### ISSUE TITLE
FR: Put power search selection on main search page

### ISSUE THREAD

**Issue by SBGit-2019** [eikek/docspell#2617]
Power search can be enabled in the settings. However, you have to decided which search you want there. It would be useful to have the power search directly on the main search page.

Here some ideas how to do it:
- Either use the arrows which now select between names and content to go to a third option "power search"
- Offer the check button for power search on the main search page
- Make a second search field for power search

**Comment by eikek** [comment]
Yes, the situation with these search options is not ideal. I wanted to change this for some time, but didn't find the time yet. Another option I thought about is have only a single search and it would detect if you typed in some `field:value` pattern or not. If not, it uses name search if there is no fulltext search configured, otherwise it uses fulltext search.
